src/app/container/container.component.html

```diff
@@ -1,138 +1,164 @@
 <div class="container">
-  <h3>{{ title }}</h3>
-  <p>
-    Last build: {{ tool?.agoMessage || 'n/a' }}
-  </p>
-</div>
+  <div class="row">
+    <h3>
+      <span id="verifiedIcon" *ngIf="tool?.versionVerified">
+        <a href="https://support.twitter.com/articles/119135#">
```

Comment:
Make an issue for us to write something up to describe verified tools so that we do not forget